src/libs/elektra/keyset.c

```diff
@@ -1191,7 +1191,7 @@ KeySet * ksCut (KeySet * ks, const Key * cutpoint)
 		++it;
 	}
 
-	// correct cursor if cursor is in cutted keyset
+	// correct cursor if cursor is in cut keyset
```

Comment:
Thank you! Maybe we could add even a rule to always replace cutted with cut?

Reply:
I do not think that does not make much sense currently, since `fix-spelling` does not check mistakes inside source code. I replaced another occurrence of “cutted” with “cut” in commit 49fd0f87.